Jenkinsfile.nightly

```diff
@@ -44,5 +44,31 @@ pipeline {
                 }
             }
         }
+
+        stage('Kubernetes tests') {
+            environment {
+                GOPATH="${WORKSPACE}"
+                TESTDIR="${WORKSPACE}/${PROJ_PATH}/test"
+            }
+            steps {
+                parallel(
+                    "K8s-1.7":{
```

Comment:
Liking the move of older K8s versions to nightly tests. What is the rationale behind keeping the 1.6 tests around as part of each PR? I'm for moving them to nightly as well.

Reply:
A lot of test has different cases for 1.6, the CNP v1, so I think that we should keep it.

Reply:
Fair enough.